src/main/java/org/sagebionetworks/web/client/widget/ReactComponentLifecycleUtils.java

```diff
@@ -0,0 +1,18 @@
+package org.sagebionetworks.web.client.widget;
+
+import org.sagebionetworks.web.client.jsinterop.ReactDOM;
+
+import com.google.gwt.dom.client.Element;
+
+/**
+ * Utility methods that manage the React component lifecycle from GWT lifecycle methods
+ */
+public abstract class ReactComponentLifecycleUtils {
```

Comment:
I needed a ReactComponentSpan without duplicating code from ReactComponentDiv. Not sure if there is a better way to do this since Java doesn't permit multiple inheritance of classes.

Reply:
Not sure if it makes sense that this is an abstract class, if this just contains a couple of static utility methods.  Recommend either a normal utility class that contain the static methods, or (if it makes sense) an abstract class that contains a couple of non-static methods that are used by the classes that implement the class.

Reply:
Definitely shouldn't be abstract! I was experimenting to see if I could do something with inheritance with ReactComponentDiv/Span and forgot to change it.